hatch/src/cli/commands/new.rs

```diff
@@ -25,7 +25,9 @@ impl<'command> Command<'command> for New {
         let dir_path = self.project_path(args).join(&name);
         let hatch_file = hatchfile_path(&dir_path);
         let includes = self.parse_dependencies(args);
-        fs::create_dir(&dir_path)?;
+        if !dir_path.exists() {
```

Comment:
This shouldnt be possible

Reply:
how? we don't create the dir until this function. We only create path buf.

Reply:
oh nvm